template.yaml

```diff
@@ -194,7 +194,7 @@ Resources:
     Properties:
       FunctionName: SungardAS-Alerts-Teams
       Handler: index_teams.handler
-      Runtime: nodejs4.3
+      Runtime: nodejs8.10
```

Comment:
Make changes to accept this as a param

Reply:
Runtime value cannot be a Parameter. still if we need to try this , then we can do while we move code to BitBucket.